azure-pipelines-official.yml

```diff
@@ -2,13 +2,33 @@ resources:
 - repo: self
   clean: true
 
-# Variables defined in yml cannot be overridden at queue time instead overrideable variables must be defined in the web gui.
-# Commenting out until AzDO supports something like the runtime parameters outlined here: https://github.com/Microsoft/azure-pipelines-yaml/pull/129
-#variables:
-#  SignType: real
-#  SkipTests: false
-#  SkipApplyOptimizationData: false
-#  IbcDrop: 'default'
+parameters:
+- name: IbcDrop
+  type: string
+  default: default
+
+- name: IbcSourceBranchName
```

Comment:
I am not certain this parameter works anymore. I've seen reference to a _IbcSourceBranchName tho.

Reply:
Perhaps arcade targets pick it up in the environment variables. I kicked this build using the variable, let's see if it has the desired effect. https://dev.azure.com/dnceng/internal/_build/results?buildId=1206551&view=results